mtvsrs/views.py

```diff
@@ -1,3 +1,24 @@
+from django.shortcuts import render, redirect
+from mtvsrs.models import Movie
+from django.contrib.auth.forms import UserCreationForm
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect
+from django.urls import reverse
+from django.shortcuts import render, redirect
+
+def register_user(request: HttpRequest) -> HttpResponse:
+
+    if request.method == "POST":
+        form = UserCreationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            success_url = reverse("index")
+            return HttpResponseRedirect(success_url)
+    else:
+        form = UserCreationForm()
+
+    context = {"form": form}
+
+    return render(request, "registration/registration_form.html", context)
```

Comment:
might as well flatten the folder, change file name to registration.html (just to keep consistent with our other templates)